azure-kusto-data/azure_bdist_wheel.py

```diff
@@ -7,13 +7,19 @@
 from distutils import log as logger
 import os.path
 
+# review: the 'wheel' package has no officially public API, so be aware that this could break at any time.
 from wheel.bdist_wheel import bdist_wheel
+# review: two newlines between top-level definitions (PEP 8).
+
+# review: class names should be CapWords (PEP 8)
```

Comment:
hum, I tried to follow the setuptools convention, so naming the azure version from "azure_bdist_wheel" from "bdist_wheel". That's pure packaging, and no one except the setup.py will use that anyway.